backend/src/core/register.py

```diff
@@ -16,9 +17,14 @@ def _init_app(version: str) -> FastAPI:
 
 
 def _read_version() -> str:
-    with open("src/VERSION.txt", "r") as version_file:
-        version = version_file.read()
-        return version
+    try:
+        with open("src/VERSION.txt", "r") as version_file:
+            version = version_file.read()
+            return version.strip()
```

Comment:
I think to this kind of methods (which are not 100% clear we can add a little docstrings) but its only suggestion.